optimum/intel/neural_compressor/optimization.py

```diff
@@ -139,6 +139,17 @@ def save_pretrained(self, save_directory: Optional[Union[str, os.PathLike]] = No
 
         os.makedirs(save_directory, exist_ok=True)
         self.config.save_pretrained(save_directory)
+        if isinstance(self._model.model, torch.jit._script.RecursiveScriptModule):
+            import json
+
+            try:
+                with open(os.path.join(save_directory, "best_configure.json"), "w") as f:
```

Comment:
Is the `best_configure` json file is used when loading a quantized model ? If not I would remove saving it

Reply:
Since `best_configure.json` contains more information than the pt file, like data type of each ops, we recommend saving the json file.